### ISSUE TITLE
jQuery Mobile 1.4

### ISSUE THREAD

**Issue by jonathannaguin** [ququplay/jquery-mobile-flat-ui-theme#14]
So jQuery Mobile people is about to release the public 1.4, right now there is a pre-release with a lot of [changes](https://github.com/jquery/jquery-mobile/wiki/1.4-changelog). Among the most important this:
- No generating of inner button markup.

might be a problem, riight?

**Comment by edasque** [comment]
@mkuklis , is the work for 1.4.x complete at this point? Is this still being maintained?

**Comment by tlkh** [comment]
@edasque Since there's been little activity and author seems to have moved on to premium themes, I've been working on my own fork of this theme for a while now.
https://github.com/tlkh/jquery-mobile-flat-ui-theme
Fully working with jQM 1.4.5, a few design tweaks, SVG icons instead of PNG (much better looking on high-res devices) and a new spinner icon.

**Comment by edasque** [comment]
@tlkh awesome, I'll take a look! Thank you

**Comment by mkuklis** [comment]
@edasque we focused our time on a new premium theme but we hope to come back and refresh this one at some point in the future. @tlkh it's nice to see your progress. Nice job!

**Comment by edasque** [comment]
@mkuklis understood. I just wouldn't buy a jQuery mobile theme without being able to test it first.

**Comment by tlkh** [comment]
@mkuklis thanks for laying all the ground work for this! Anticipating your future work indeed.

**Comment by mkuklis** [comment]
@edasque I agree I wouldn't either. If you go to our store http://store.ququplay.com there is a preview of the theme. @tlkh I'm glad you are taking it further. Cheers!
